src/components/RangeInput.vue

```diff
@@ -1,18 +1,16 @@
 <template>
 <div v-if="state" :class="suit()">
   <slot
-    :updateMin="this.updateMin"
-    :updateMax="this.updateMax"
     :currentRefinements="this.state.start"
     :refine="this.refine"
   >
-  <form :class="suit('form')" @submit.prevent="refine()">
+  <form :class="suit('form')" @submit.prevent="refine(minInput, maxInput)">
     <label :class="suit('label')">
-      <input :class="[suit('input'), suit('input', 'min')]" type="number" :min="this.min"  :placeholder="this.min" :value="this.state.start && this.state.start[0]" @change="updateMin($event.currentTarget.value)"/>
+      <input :class="[suit('input'), suit('input', 'min')]" type="number" :min="this.min"  :placeholder="this.min" :value="this.state.start && this.state.start[0]" @change="minInput = $event.currentTarget.value"/>
```

Comment:
last Sunday I discovered `valueAsNumber` is a thing! https://www.w3.org/TR/2011/WD-html5-20110405/common-input-element-attributes.html#dom-input-valueasnumber it's not supported on IE11 though for example :(

Reply:
That's amazing! :) Should I implement it then, even with ie11?

Reply:
no, you should probably not use it, because the support is hard to find, while the value is fairly minimal